src/patterns/InPlaceFilters/FilterButton.tsx

```diff
@@ -0,0 +1,204 @@
+import * as Popover from '@radix-ui/react-popover';
+import classnames from 'classnames';
+import React from 'react';
+import Button from '../../components/Button/Button';
+import { ButtonVariants } from '../../components/Button/types';
+import Drawer from '../../components/Drawer/Drawer';
+import Filter from '../../components/Filter/Filter';
+import FilterHeader from '../../components/Filter/FilterHeader';
+import FilterInput from '../../components/Filter/FilterInput';
+import { SSRMediaQuery } from '../../providers/SeldonProvider/utils';
+import { px } from '../../utils';
+import FilterMenu from '../FilterMenu/FilterMenu';
+import { FilterButtonDisplay } from './FilterButtonDisplay';
+import { FilterDropdown } from './FilterDropdown';
+import { FilterButtonProps, FilterDimension, FilterType } from './types';
+import { countActiveFilters } from './utils';
+
+export const FilterButton = React.forwardRef<HTMLButtonElement, FilterButtonProps>(
+  (
+    {
+      id,
+      className,
+      filterButtonLabel,
+      buttonType,
+      handleClick,
+      filtersListState,
+      index,
```

Comment:
can we call `index` an `id` instead and have a unique ID for each filter dimension instead.  `Index` implies it might change if the order changes for some reason